foundationdb/src/tuple/mod.rs

```diff
@@ -189,13 +212,13 @@ mod tests {
 
     #[test]
     fn test_malformed_int() {
-        assert!(Tuple::decode(&[21, 0]).is_ok());
-        assert!(Tuple::decode(&[22, 0]).is_err());
-        assert!(Tuple::decode(&[22, 0, 0]).is_ok());
+        assert!(Tuple::decode(&[21, 0], 0).is_ok());
```

Comment:
Could we use `try_from` instead of `decode`, as following tests do?

Reply:
sure, I can make that change.